Sources/NostrSDK/Events/Tags/EventCoordinates.swift

```diff
@@ -9,7 +9,7 @@ import Foundation
 
 /// Coordinates to a (maybe parameterized) replaceable event.
 /// See [NIP-01 Tags](https://github.com/nostr-protocol/nips/blob/master/01.md#tags).
-public struct ReplaceableEventCoordinates: PubkeyProviding, RelayProviding, RelayURLValidating, Equatable {
+public struct EventCoordinates: PubkeyProviding, RelayProviding, RelayURLValidating, Equatable {
```

Comment:
I decided that the `Replaceable` part of the naming was too lengthy and not part of the wording in the spec, so I removed it.